graylog2-server/src/main/java/org/graylog/security/certutil/keystore/storage/KeystoreStorage.java

```diff
@@ -32,4 +32,5 @@ void writeKeyStore(final T location,
 
     Optional<KeyStore> readKeyStore(final T location, char[] password) throws KeyStoreStorageException;
 
+    void removeKeyStore(final T location);
```

Comment:
I'd remove this method from interface. It suggest false assumption - that each storage supports removal.